pom.xml

```
@@ -12,28 +12,30 @@
 	<url>https://github.com/deviceinsight/helm-maven-plugin</url>
 
 	<properties>
+		<!-- Compiler -->
 		<kotlin.version>1.8.10</kotlin.version>
 		<kotlin.compiler.jvmTarget>1.8</kotlin.compiler.jvmTarget>
 		<maven.compiler.source>1.8</maven.compiler.source>
 		<maven.compiler.target>1.8</maven.compiler.target>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
+		<!-- Maven -->
 		<maven.version>3.6.2</maven.version>
-		<maven-project.version>2.2.1</maven-project.version>
+		<maven-plugin-tools.version>3.8.1</maven-plugin-tools.version>
 
+		<!-- Project dependencies -->
 		<jackson-bom.version>2.14.2</jackson-bom.version>
 		<httpclient.version>4.5.14</httpclient.version>
 		<plexus-sec-dispatcher.version>1.4</plexus-sec-dispatcher.version>
 
-		<junit.jupiter.version>5.9.2</junit.jupiter.version>
-		<assertj-core.version>3.24.2</assertj-core.version>
-
-		<dokka-maven-plugin.version>1.7.20</dokka-maven-plugin.version>
+		<!-- Plugins -->
 		<maven-compiler-plugin.version>3.11.0</maven-compiler-plugin.version>
 		<maven-source-plugin.version>3.2.1</maven-source-plugin.version>
 		<gitflow-maven-plugin.version>1.19.0</gitflow-maven-plugin.version>
 		<nexus-staging-maven-plugin.version>1.6.13</nexus-staging-maven-plugin.version>
 		<maven-gpg-plugin.version>3.0.1</maven-gpg-plugin.version>
+		<maven-invoker-plugin.version>3.3.0</maven-invoker-plugin.version>
+		<maven-invoker-plugin.groovy.version>4.0.6</maven-invoker-plugin.groovy.version>
```

Let's not add dependencies already out of date…
```suggestion
		<maven-invoker-plugin.version>3.5.0</maven-invoker-plugin.version>
		<maven-invoker-plugin.groovy.version>4.0.10</maven-invoker-plugin.groovy.version>
```